client/src/components/TaskCard/TaskCard.module.css

```diff
@@ -0,0 +1,45 @@
+.taskContainer {
```

Comment:
I would also consider adding another CSS class such that when the task is completed, a strikethrough or something that's easily noticeable appears to the user to indicate that the task is complete.

Reply:
Added strikethrough for tasks marked done.

Reply:
![image](https://user-images.githubusercontent.com/24628243/157380004-4cad08f8-6e42-4391-8841-0d1cbcdf49e3.png)

I would perhaps make the strikethrough with perhaps a more visible color. Although the completed tasks do have an off-white color compared to the current tasks, I think it should be a bit more visible